rpcserver.go

```diff
@@ -286,10 +286,29 @@ func (r *rpcServer) DebugLevel(ctx context.Context,
 func (r *rpcServer) GetInfo(context.Context,
 	*taprpc.GetInfoRequest) (*taprpc.GetInfoResponse, error) {
 
+	// Retrieve the best block hash and height from the chain backend.
+	blockHash, blockHeight, err := r.cfg.Lnd.ChainKit.GetBestBlock(context.Background())
```

Comment:
We should use the inbound context (just need to give it and the `*taprpc.GetInfoRequest` a parameter name). Here and below.